packages/components/pie-radio-group/src/index.ts

```diff
@@ -147,12 +160,152 @@ export class PieRadioGroup extends FormControlMixin(RtlMixin(LitElement)) implem
         }
     }
 
+    protected firstUpdated (): void {
+        // Make all radios impossible to tab to
```

Comment:
It'd be nice to add to the comment why it is necessary.

Reply:
I can update that now - thanks!